classes/opponentPhasePortals.lua

```diff
@@ -0,0 +1,78 @@
+-- Now you're thinking with portals
+
+local opponentPhasePortals = {}
+opponentPhasePortals.__index = opponentPhasePortals
+
+
+--------------------
+-- MAIN CALLBACKS --
+--------------------
+
+function opponentPhasePortals.new()
+	local self = classes.opponentBase.new(opponentPhasePortals)
+
+	self.CHASE_SPEED = 200							--Is there a danger in having the same variable names across different phases/files?
```

Comment:
No there isn't. If it's inside a `self.` block then it's local to that object. If it has `local` at the top of the file, the it's local to this specific file. If there is no `self.` or `local`, then it is a global variable and it can be dangerous.

Reply:
Might be good to delete the comment in the code btw